locales/en-US/app.ftl

```diff
@@ -257,6 +257,11 @@ Home--menu-button-instructions =
     Enable the profiler menu button to start recording a performance
     profile in { -firefox-brand-name }, then analyze it and share it with profiler.firefox.com.
 
+Home--profile-firefox-android-instructions =
+    You can also profile Firefox for Android. For more
```

Comment:
```suggestion
    You can also profile { -firefox-brand-name } for Android. For more
```

Or we can add a brand name for `Firefox for Android`.

Reply:
I didn't want to replace with Nightly on Nightly, because the user isn't especially using Nightly on Android if they're using Nightly on Desktop.

But now I realize that the brand name is fixed in the Profiler ftl file anyway... So why not

What's the value of having a brand name in our case BTW?

Reply:
> What's the value of having a brand name in our case BTW?

Consistency, and it's easy to check in only one place (string) that the brand rules are respected.

Reply:
I added a new brand name, thanks for your feedback